components/RadioGroup/RadioButton.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import ButtonGroupLabel from '../shared/ButtonGroupLabel';
+import HiddenRadio from '../shared/HiddenRadio';
+import Icon from '../Icon';
+import uniqId from '../shared/uniqId';
+
+const LockIcon = styled(Icon).attrs({
+  name: 'lock',
+})``;
+
+const ButtonIcon = styled(Icon)`
+  margin-right: 8px;
+`;
+
+const Wrapper = styled.div`
+  ${({ inline }) =>
+    inline &&
+    `
+    display: inline-block;
+    margin-right: 16px;
+    vertical-align: top;
+
+    :last-child {
+      margin-right: 0;
+    }
+  `}
+
+  ${LockIcon} {
+    font-size: 17px;
+    margin-left: 8px;
+  }
+`;
+
+const Radio = ({
+  children,
+  label,
+  error,
+  disabled,
+  onChange,
+  value,
+  checked,
+  icon,
+  id,
+  inline,
+  ...rest
+}) => {
+  const skin = checked ? 'primary' : 'secondary';
+  const _id = id || uniqId('radio-button-');
```

Comment:
Still about generators, this line would look somethings like this:

```suggestion
const _id = id || ID_GENERATOR.next().value;
```

Reply:
Done